buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/TestProgressLogger.groovy

```diff
@@ -49,21 +53,81 @@ class TestProgressLogger implements AggregatedEventListener {
         progressLogger = factory.newOperation(TestProgressLogger)
         progressLogger.setDescription('Randomized test runner')
         progressLogger.started()
-        progressLogger.progress('Starting JUnit4 with ' + e.getSlaveCount() + ' jvms')
+        progressLogger.progress("Starting JUnit4 with $e.slaveCount jvms")
```

Comment:
`${}` gives me flashbacks to maven. I don't personally think its more clear but I'll do it because you do.